### PR TITLE
Package adafruit-pitft-helper, a variant of pitftupdater.sh

### PR DESCRIPTION
**Pull request by brennen**
More complicated than it needs to be, but simple in outcome:  A package that adds `adafruit-pitft-helper` script to path and installs its dependencies.

Will become considerably more useful with a dependency on a kernel package (coming soon to an apt repo near you).

### CODE REVIEW

**Comment by toddtreece** [comment]
should `packages/adafruit-pitft-helper_TEMPLATE/debian/adafruit-pitft-helper.debhelper.log` be ignored?

**Comment by brennen** [comment]
Good catch. I think probably yes.

File: packages/adafruit-pitft-helper_TEMPLATE/debian/files [hunk 1]
```diff
@@ -0,0 +1 @@
+adafruit-pitft-helper_0.0.1-1_armhf.deb utils extra
```
Comment (toddtreece) [inline packages/adafruit-pitft-helper_TEMPLATE/debian/files]: i think i also read somewhere that this file shouldn't be in version control, but i can't find the source of that info after searching for 15 mins.

File: packages/Makefile [hunk 2]
```diff
@@ -15,3 +20,18 @@ build/occidentalis_${OCCIDENTALIS_VERSION}_armhf.deb: ${occidentalis_stuff}
...
+# I'm sorry
+build/adafruit-pitft-helper_${PITFT_PACKAGE_VERSION}_armhf.deb: ${pitft_stuff}
+	mkdir -p adafruit-pitft-helper
+	rm -r adafruit-pitft-helper
+	mkdir adafruit-pitft-helper
+	cd adafruit-pitft-helper && wget https://github.com/adafruit/Adafruit-PiTFT-Helper/archive/master.tar.gz -O adafruit-pitft-helper_${PITFT_VERSION}.orig.tar.gz
+	cd adafruit-pitft-helper && tar xvzf adafruit-pitft-helper_${PITFT_VERSION}.orig.tar.gz
+	cd adafruit-pitft-helper && mv Adafruit-PiTFT-Helper-master adafruit-pitft-helper_${PITFT_VERSION}
+	cp -r adafruit-pitft-helper_TEMPLATE/debian adafruit-pitft-helper/adafruit-pitft-helper_${PITFT_VERSION}/
+	cd adafruit-pitft-helper/adafruit-pitft-helper_${PITFT_VERSION} && debuild -b -aarmhf -uc -us
```
Comment (toddtreece) [inline packages/Makefile]: does this build ok without specifying a cross compiler, or are you building the deb on a Pi? maybe there's some build dependency i'm missing.

Comment (brennen) [inline packages/Makefile]: There's really nothing to compile - it's just a shell script + dependencies. Maybe arch should be any, although it's going to have _module_-specific dependencies that in practice mean "this runs on a pi", right?

Comment (toddtreece) [inline packages/Makefile]: ah `debuild` was blowing up without the proper compiler installed for the kernel debian stuff, and it doesn't build anything. maybe it's doing something else

**Pull request merged**